scripts/build-retention/monitoring/alert.rules

```diff
@@ -0,0 +1,42 @@
+groups:
+- name: alert.rules
+  rules:
+  - alert: cortx-storage
+# Change all ips below to that of the mounted node.
+    expr: 100 - (100 * ((node_filesystem_avail_bytes{device="cortx-storage.colo.seagate.com:/mnt/data1", fstype="nfs4", instance="10.230.242.163:9100", job="node-exporter", mountpoint="/rootfs/mnt/cortx-storage"} )  / (node_filesystem_size_bytes{device="cortx-storage.colo.seagate.com:/mnt/data1", fstype="nfs4", instance="10.230.242.163:9100", job="node-exporter", mountpoint="/rootfs/mnt/cortx-storage"}) )) > 95
```

Comment:
please hardcoded IP's from this file

Reply:
IP is reference to build-retention node where deployment is running & LCO NFS is mounted. Developer will change IP on deploying to next node. Current IP shows deployed node and should be treated as placeholder reference.